src/indra_cogex/representation.py

```diff
@@ -497,11 +503,21 @@ def indra_stmts_from_relations(rels: Iterable[Relation],
     """
     stmts_json = [load_statement_json(rel.data["stmt_json"]) for rel in rels]
     stmts = stmts_from_json(stmts_json)
-    # Beliefs are not set correctly in the JSON so we fix them here
+
+    # Beliefs are not set correctly in the JSON, so we fix them here
```

Comment:
I think it's actually fixed now.

Reply:
Do you want me to take it out or leave it as it is? Since it was already present earlier.

Reply:
Yes, I just verified this: the database now contains the same belief value in the relationship property and inside the statement json data associated with the relationship. So we can remove the part that corrects for this.